### PR TITLE
Fix tests against Wagtail 4.x

### PR DESCRIPTION
**Pull request by gasman**

### CODE REVIEW

**Review by Stormheg — APPROVED** [review]
All looks sensible to me.

File: wagtail_review/migrations/0001_initial.py [hunk 1]
```diff
@@ -4,7 +4,24 @@
...
 from django.conf import settings
 from django.db import migrations, models
+from django.db.migrations.recorder import MigrationRecorder
 import django.db.models.deletion
+from wagtail import VERSION as WAGTAIL_VERSION
+
+
+def get_revision_model():
+    revision_model = "wagtailcore.PageRevision"
+    if WAGTAIL_VERSION >= (4, 0, 0):
```
Comment (zerolab) [inline wagtail_review/migrations/0001_initial.py:14]: I missed one edge case here.
See https://github.com/wagtail/wagtail-localize/pull/643 for the updated version

**Comment by oladhari** [comment]
any progress to merge this PR and create new release?

File: wagtail_review/migrations/0001_initial.py (hunk 1 above)
Comment (gasman) [reply wagtail_review/migrations/0001_initial.py:16]: Thanks @zerolab! Have now ported this fix in 8763abe24d7fe1589d279031a4c517f01be4d104.

**Review by zerolab — APPROVED** [review]
LGTM

**Comment by gasman** [comment]
Merged in 2dd933da69137cc66f81fe001d20e9db69ff2b22

**Pull request closed without merging**